server/src/test/scala/za/co/absa/atum/server/api/database/runs/functions/WriteCheckpointIntegrationTests.scala

```diff
@@ -53,7 +54,7 @@ object WriteCheckpointIntegrationTests extends ConfigProviderTest {
     ).provide(
       WriteCheckpoint.layer,
       PostgresDatabaseProvider.layer,
-      TestTransactorProvider.layerWithRollback
+      TestTransactorProvider.layerWithoutRollback
```

Comment:
Please add back the layer with rollback. These are the changes you have made when we were debugging the code on the call, but it should be reverted.